spec/system/submissions/review_spec.rb

```diff
@@ -1075,7 +1075,6 @@
       expect(page).to have_content(evaluation_criterion_2.name)
       expect(page).to have_button('Save grades', disabled: true)
       click_button 'Remove assignment'
-      dismiss_notification
```

Comment:
_Remove assignment_ will fire a notification, `dismiss_notification` is required in this case

Reply:
@bodhish Transient failures were occurring on this test because of this line. Since no action is taken after this particular notification is created, there is no real need to dismiss it (its presence will not occlude any further action).